read_lyrics.py

```diff
@@ -0,0 +1,105 @@
+import pronouncing
+
+def analyze_text(myString):
+    myString = myString.replace(",", "")
+    myString = myString.replace(".", "")
+    myString = myString.replace("?", "")
+    myList = myString.split(" ")
+    running = []
+    running.append(myList[0])
+
+    idk = 0
+    same = 0
+    usedRhymes = []
+    for i in range(1, len(myList)):
+        for j in running:
+            if not usedRhymes.__contains__(j + myList[i]):
+                usedRhymes.append(j + myList[i])
+                rating = two_rhyme(j, myList[i])
+                if rating == -1:
+                    while two_rhyme(j, myList[i]) == -1 and len(myList[i]) > 1:
+                        myList[i] = myList[i][1:len(myList[i])]
+                    if two_rhyme(j, myList[i]) == -1:
+                        idk += 1
+                        break
+                if rating >= 100:
+                    same += 1
+                    print(j + ' ' + myList[i])
+
+        running.append(myList[i])
+        if len(running) > 15:
+            running.pop(0)
+
+    print(idk)
+    rate = same / len(myList)
+    return same, len(myList)
+
+
+def two_rhyme(Sinit, Sinit2):
+    St1 = Sinit.lower()
```

Comment:
Docstring containing a brief description of the function, what inputs it is expecting, and what output it produces would be appropriate here.